### PR TITLE
32 page update time is just this with some extra stuff

### PR DESCRIPTION
**Pull request by yyy2356**
might have committed some stuff without doing a checkout first. my bad. Just going to merge these two, i havent found any issues with this, so it should probably be good to merge

### CODE REVIEW

File: HeroHours/views.py [hunk 1]
```diff
@@ -144,15 +146,21 @@ def handle_bulk_updates(user_id):
...
+    change = "None"
+    orig = user.Total_Seconds
     if user.Checked_In:
         count2 -= 1
         state = False
         log.operation = 'Check Out'
         user.Total_Hours = ExpressionWrapper(F('Total_Hours') + (right_now - user.Last_In),
                                              output_field=DurationField())
         user.Total_Seconds = F('Total_Seconds') + round((right_now - user.Last_In).total_seconds())
+        #print(round((right_now - user.Last_In).total_seconds()))
```
Comment (MichaelStatt-S) [inline HeroHours/views.py:162]: maybe I'm too persnickety, but can you remove these extra commented out prints?

File: HeroHours/views.py [hunk 2]
```diff
@@ -144,15 +146,21 @@ def handle_bulk_updates(user_id):
...
+        change = round((right_now - user.Last_In).total_seconds())
```
Comment (MichaelStatt-S) [inline HeroHours/views.py:163]: Am I missing something? Can't you just use the existing totalsecound logic?

**Review by MichaelStatt-S — CHANGES_REQUESTED** [review]

**Comment by MichaelStatt-S** [comment]
Why are you merging into 21-make-logout-command-in-page? why not into main

File: HeroHours/views.py [hunk 3]
```diff
@@ -180,9 +189,20 @@ def check_in_or_out(user, right_now, log, count):
         'state': state,
         'newlog': model_to_dict(log),
         'count': count,
+        'newtime': logged_time,
     }
 
 
+def get_time(orig: float, change: int) -> str:
```
[the remark this answers is not in the document]
Comment (yyy2356) [reply HeroHours/views.py:196]: > Why isn't this just using the function that's already in the model?
when making changes to data, the change is stored as a code block in memory, essentially. to use the built in method on the changed data would require a significant change to the built in method or to save and then re access the data for the user. The first is a problem for obvious reasons, the second is really inefficient, so i opted to do this.

**Comment by yyy2356** [comment]
made a mistake and didnt checkout, so the changes are the same on both, except for the most recent 2 commits, where i fix a issue with the login redirect url, not sure when that got in there.

**Comment by yyy2356** [comment]
image

**Review by MichaelStatt-S — APPROVED** [review]
Looks good to me!

**Pull request merged**